src/AltinnCore/Designer/Controllers/PipelinesController.cs

```diff
@@ -0,0 +1,88 @@
+using System.Threading.Tasks;
+using AltinnCore.Designer.Repository.Models;
+using AltinnCore.Designer.Services;
+using AltinnCore.Designer.TypedHttpClients.AzureDevOps;
+using AltinnCore.Designer.TypedHttpClients.AzureDevOps.Models;
+using AltinnCore.Designer.ViewModels.Request;
+using Microsoft.AspNetCore.Mvc;
+
+namespace AltinnCore.Designer.Controllers
+{
+    /// <summary>
+    /// Controller for pipelines
+    /// </summary>
+    [ApiController]
+    [Route("/designer/api/v1/")]
+    public class PipelinesController : ControllerBase
+    {
+        private readonly IAzureDevOpsBuildService _buildService;
+
+        /// <summary>
+        /// Constructor
+        /// </summary>
+        public PipelinesController(
+            IAzureDevOpsBuildService buildService)
+        {
+            _buildService = buildService;
+        }
+
+        /// <summary>
+        /// Gets a build status from Azure DevOps and updates a specific entity
+        /// </summary>
+        /// <returns>Created release</returns>
```

Comment:
Doesn't seem to actually return a created release? Should the comment be updated or are we missing something in the return ?

Reply:
Updating, good catch!